src/index.html

```diff
@@ -15,8 +15,104 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo__link"
+      >
+        <img
+          class="logo__img"
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
```

Comment:
All navigation links currently point to 'index.html'. If these links are intended to navigate to different pages or sections, consider updating the 'href' attributes to the correct URLs.